docker/builder/build-tools.sh

```diff
@@ -45,5 +45,6 @@ for BIN in "${BINS[@]}"; do
 done
 
 # Build the Aptos Move framework and place it in dist. It can be found afterwards in the current directory.
+echo "MOVE_COMPILER_V2: ${MOVE_COMPILER_V2:-not set}"
```

Comment:
Is there a reason we dont just set this here to ensure that its set whenever we're running tools?c

Reply:
We can make the change here instead

I originally planned to have two tools image with compiler v2 enabled/disabled so I put this in the Dockerfile

Reply:
I'm just going to keep the env var in the Dockerfile since this is already tested. This looks like the only place we use the build-tools script